app/views/smart_proxies/plugins/_plugin_version.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-md-4"><%= _('Plugin version') %></div>
```

Comment:
I don't think this should show up if the feature doesn't come from a plugin. Just `version` is fine or even better, nothing when the feature is part of the default features in the proxy, and 'plugin version' when it comes from a plugin.